homeassistant/components/androidtv/media_player.py

```diff
@@ -475,6 +477,32 @@ def unique_id(self):
         """Return the device unique id."""
         return self._unique_id
 
+    async def async_get_media_image(self):
+        """Fetch current playing image."""
+        media_data = self.get_raw_media_data()
+        if media_data:
+            return binascii.a2b_base64(media_data), "image/png"
+        return None, None
+
+    def get_raw_media_data(self):
+        """Raw base64 image data."""
+        if self.state not in [STATE_OFF, None]:
+
+            try:
+                response = self.aftv.adb_shell("screencap -p | base64")
+            except UnicodeDecodeError:
+                return None
+
+            if isinstance(response, str) and response.strip():
+                return response.strip().replace("\n", "")
+
+        return None
+
+    @property
+    def media_image_url(self):
+        """URL for obtaining a screen capture."""
+        return f"{datetime.now().timestamp()}"
```

Comment:
This should be valid url if you have one. This is not it. You should implement `media_image_hash`

Reply:
@balloob do we want to retrieve screenshots and use them as media art?

Reply:
Yeah if it's possible and it doesn't slow down the device, why not.

Reply:
@balloob I did notice the media_image_hash ... however not all integrations use this ... Also there is no valid URL ... but without specifying one that gets updated, async_get_media_image never gets called, meaning that the image updates.

Reply:
@JeffLIrion It doesn't seem to make any noticeable difference in speed... I have an LG Netcast TV and that integration returns screen dumps for the Album Art.

Reply:
> not all integrations use this

It's only used internally by the media player base entity, which uses it.

Reply:
@balloob .. So do I do something with this when there is no valid URL?

Reply:
just return a timestamp from hash and you should be fine.

Reply:
Ok ... I'll check if that works and re-commit

Reply:
Works ... but the hash I left as the timestamp ... since the value seems somewhat arbitrary... as long as it updates... re-committed.

Reply:
yeah that's fine.

Reply:
The reason the hash exists is so the browser won't cache the picture.